libs/core/.eslintrc.json

```diff
@@ -7,6 +7,8 @@
   "plugins": ["@typescript-eslint"],
   "rules": {
     "@stencil/decorators-context": "off", // This is set to off until support for ESLINT 8, othwerise remove rule
-    "@stencil/strict-boolean-conditions": "off" // This is set to off until support for ESLINT 8, othwerise remove rule
+    "@stencil/strict-boolean-conditions": "off", // This is set to off until support for ESLINT 8, othwerise remove rule
+    "@stencil/decorators-style": "off",
+    "@stencil/own-methods-must-be-private": "off"
```

Comment:
Can you explain the need for adding these?

Reply:
Most methods can be private with no issue, however, there is a problem with linting not understanding that methods attached to a `@Listen` or `@Event` decorator will be invoked automatically when those decorators are called. 

When these methods are private, the linter is smart enough to "know" the method hasn't been explicitly invoked in that TSX file which his throws a warning about the method never being used. If the method is not private then the linter doesn't throw an error about the method never being used. 

I'm happy to solve this issue in a different way if you have any thoughts on a better approach. I do think its best for us to mark most of our methods as private.